### PR TITLE
[MultiDB] use get API to obtain dbid instead of hardcode value

### PR DESCRIPTION
**Pull request by dzhangalibaba**
* use get API to obtain dbid instead of hardcode value

Signed-off-by: Dong Zhang d.zhang@alibaba-inc.com

### CODE REVIEW

**Comment by dzhangalibaba** [comment]
@kcudnik   please take a look , we can get DB id from db connector instead of hardcode COUNTERS_DB(2) . Please make sure the db is always a counters db connector, my understanding this is true.

**Comment by dzhangalibaba** [comment]
@qiluo-msft  please take a look as well

File: syncd/FlexCounter.cpp [hunk 1]
```diff
@@ -1454,7 +1454,7 @@ void FlexCounter::runPlugins(
 
     const std::vector<std::string> argv =
     {
-        std::to_string(COUNTERS_DB), // TODO
+        std::to_string(db.getDbId()),
```
Comment (lguohan) [inline syncd/FlexCounter.cpp:1457]: it db is always counter db, can you rename it to cdb?

Comment (dzhangalibaba) [reply syncd/FlexCounter.cpp]: sure. updated

**Comment by dzhangalibaba** [comment]
@kcudnik  please take a look

**Pull request merged**